pkgs/by-name/ut/utpm/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  openssl,
+  pkg-config,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "utpm";
+  version = "dev-0363e5";
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "typst-project-0.1.0" = "sha256-Qys3F6Maq0HpC/50yyAZRpsOQ+ePzaz/TuHdfjtQRwY";
+    };
+  };
+
+  src = fetchFromGitHub {
+    owner = "Thumuss";
+    repo = pname;
+    rev = version;
```

Comment:
Returns 404

Reply:
I would recommend to just use the latest commit and the 0-unstable-YYYY-MM-DD version scheme.